README.md

```diff
@@ -73,13 +73,17 @@ There is a Google Group used for project wide announcements and development coor
 Anybody can join the group by visiting [here](https://groups.google.com/forum/#!forum/cert-manager-dev)
 and clicking "Join Group". A Google account is required to join the group.
 
-Once you have become a member, you should receive an invite to the weekly development
+### Bi-weekly development meeting
+Once you have become a member, you should receive an invite to the bi-weekly development
 meeting, hosted on **Wednesdays at 4pm UTC** on Zoom.us.
```

Comment:
I think we also recently agreed to pin this to "5pm UK time" because it kept moving and confusing us. Happy to look at adjusting it (as I know that's late for you!) but let's put the 'current state of affairs' here and we can amend from there 😄

Thanks for making these updates! This is an often neglected file 😬